README.md

```diff
@@ -1,2 +1,74 @@
-# capstone-project-1
-HTML/CSS &amp; JavaScript capstone project
+# HTML/CSS & JavaScript capstone project - Conference page 🏅!
+
+Online website for the open BIM global conference. This project followed the design guidelines from 
+
+![CC Global Summit 2015 design by Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015])
```

Comment:
- [ ] Please, try to fix this broken image
<img width="448" alt="Screen Shot 2022-04-15 at 00 03 21" src="https://user-images.githubusercontent.com/49064106/163467065-8f8327ff-a87a-431f-adc9-7b591500c0f7.png">

Reply:
✔️ well resolved